demo.yaml/aperture.yaml

```diff
@@ -0,0 +1,5 @@
+%YAML 1.2
+---
+class: ApertureMockup.ApertureMockup
+configuration:
+  diameter_size_list: '[10, 20, 50, 100, 150]'
```

Comment:
In principle it is configurable, so something could be put in the configuration file instead. The hard coding is more for the test - to be sure that we are using well defined values.
This is how it would look if you have an xml file configuration:
```
 <object class="ApertureMockup">
   <username>aperture</username>
   <values>{"A5": (5, 0.11), A10": (10, 0.15), "A20": (20, 0.3), "A30": (30, 0.63), "A50": (50, 0.9), "A100": (100, 1)}</values>
 </object>
```
I guess we could try to put this in the configuration file as the ApertureMockup inherits from AbstractNState, but we should be sure to put the above values in order to satisfy the test.
I guess the yml file woul be something like this:

```
class: ApertureMockup.ApertureMockup
configuration:
  values: {"A5": (5, 0.11), A10": (10, 0.15), "A20": (20, 0.3), "A30": (30, 0.63), "A50": (50, 0.9), "A100": (100, 1)}
```
  It would be great if you could tell me if using a configuration file works for you.